api/resolvers/search.js

```diff
@@ -510,7 +510,7 @@ export default {
         item.searchTitle = e.highlight?.title?.[0] || e.highlight?.['title.exact']?.[0] || item.title
 
         // prefer the exact highlight for text
-        const searchTextHighlight = [...(e.highlight?.['text.exact'] || []), ...(e.highlight?.text || [])]
+        const searchTextHighlight = [...(e.highlight?.['text.exact'] || e.highlight?.text || [])]
```

Comment:
You don't need all the spreads if you're just taking one.

Reply:
done